### PR TITLE
Feature/jjdd8 hfv 25

### PR DESCRIPTION
**Pull request by mich-chmielewski**
web applictaion structure

### CODE REVIEW

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Comment by mich-chmielewski** [comment]
Added dbStructure from task 27

File: app/target/classes/log4j2.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (rafalmisiak) [inline app/target/classes/log4j2.xml]: Katalog target nie powinien znajdować się w repo

File: web/pom.xml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    <name>high-five-web</name>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>13</maven.compiler.source>
+        <maven.compiler.target>13</maven.compiler.target>
+    </properties>
+
+    <dependencies>
+        <dependency>
```
Comment (rafalmisiak) [inline web/pom.xml]: w zależnosciach powinna być jarka z app'a

File: web/pom.xml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.11</version>
```
Comment (rafalmisiak) [inline web/pom.xml]: nie używasz junit 4 wiec nie ma sensu go tutaj zamieszczać

File: web/target/high-five-web/index.jsp [hunk 4]
```diff
@@ -0,0 +1,5 @@
+`<html>
```
Comment (rafalmisiak) [inline web/target/high-five-web/index.jsp]: wkradł Ci się apostrof na początku pliku

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

File: app/target/classes/log4j2.xml (hunk 1 above)
Comment (rafalmisiak) [reply app/target/classes/log4j2.xml]: katalog nadal w repo

**Comment by isacademyci** [comment]
Build triggered for merge commit.

**Comment by isacademyci** [comment]
Build started for merge commit.

**Comment by isacademyci** [comment]
Build finished.

**Pull request merged**